### DISCUSSION TITLE
Simpel or no authentication

### DISCUSSION THREAD

**Discussion by neelusR** [Kong/insomnia#7292] [Ideas]
It would be great if I could start my Insomnia app like a regular app, keeping my credentials and not be hassled to authenticate by opening the website logging in then logging into my mail account, copying the authentication code, pasting my authentication code and then can finally start work.
If authentication is so necessary, please just allow me to login with credentials instead of having to open my email and fetch a code.

**Comment by subnetmarco** [comment]
Hi @neelusR we have fixed an issue in 9.0 (beta [available now](https://github.com/Kong/insomnia/releases/tag/core%409.0.0-beta.4)) where if you close the app we don't require to login again anymore.

We are also making E2EE optional, which will make Insomnia work like a "regular app" without having to enter a passphrase every time. This will also ship this month, and you will be able to disable E2EE to have a more user-friendly experience.

**Comment by neelusR** [comment]
Hi,

 

Thanks a lot, sounds good. I like Insomnia a lot, so great that this will be easier :-)

 
 

Sent: Monday, April 22, 2024 at 8:27 PM

**Comment by subnetmarco** [comment]
We are making further improvements and 9.0 will ship this week. I will close this discussion for now, but please re-open it if you have more suggestions after 9.0 has been released. Thanks.
